tests/fixtures/modifiers.js

```diff
@@ -167,6 +167,21 @@ const modifiersFixtures = [
 		'expected': '([Aa](?:a))',
 		'expectedFlags': ''
 	},
+	{
+		'pattern': '\\p{Lowercase_Letter}(?-i:k)',
+		'flags': 'iu',
+		'options': { unicodePropertyEscapes: 'transform', modifiers: 'transform' },
+		'matches': ['ck', 'Ck', 'δk', 'Δk', '\u{118A8}k', '\u{118C8}k'],
+		'nonMatches': ['cK', 'CK', 'δK', 'ΔK', '\u{118A8}K', '\u{118C8}K', 'c\u212A', 'C\u212A'],
+		'expectedFlags': 'u'
+	},
+	{
+		'pattern': '[\\p{Lowercase_Letter}&&\\p{ASCII}](?-i:a)',
+		'flags': 'iv',
+		'options': { unicodeSetsFlag: 'transform', unicodePropertyEscapes: 'transform', modifiers: 'transform' },
+		'expected': '[A-Za-z\\u017F\\u212A](?:a)',
```

Comment:
Per https://github.com/tc39/proposal-regexp-v-flag/issues/30, `\u017F` and `\u212A` should not have been included here as they are not in the simple case closure of the given set `[\\p{Lowercase_Letter}&&\\p{ASCII}]`, i.e. `(\u212A)` does not have simple case folding although `k`'s simple case folding includes `\u212A`.

I think this should be addressed in a new PR, we can introduce a new SCC case mapping and hopefully this approach may fix #66 as well.